src/app/content/components/Page.spec.tsx

```diff
@@ -95,6 +99,7 @@ describe('Page', () => {
         text
         <a href="">link with empty href</a>
         <a href="#hash">hash link</a>
+        <a href="?archive=some-content">archive link</a>
```

Comment:
instead of adding this link, you should be looking to see that "when the archive url config is set, the query parameter is added to the links on lines 93 and 95"